dotnet/cloudbuild.yaml

```diff
@@ -0,0 +1,14 @@
+# Cloud builder for the dotnet builder.
+steps:
+- name: gcr.io/cloud-builders/docker
+  args: [ 'build', '-t', 'gcr.io/$PROJECT_ID/dotnet', '.' ]
+
+# Print versions information
```

Comment:
Is this to ensure that it's processing the config files properly? Can you make a step that checks that output?

```
  name: 'gcr.io/$PROJECT_ID/dotnet'
  entrypoint: 'bash'
  dir: 'configs/config-1.0.0'
  args: ['-c', '[[ $(dotnet --version) == "expected output" ]]']
```